playbooks/roles/caasp4/tasks/setup_nodes.yml

```diff
@@ -0,0 +1,87 @@
+---
+#
+# (c) Copyright 2019 SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- name: Copy skuba openstack terraform files to socok8s workspace
+  copy:
+    src: "{{ playbook_dir }}/../submodules/skuba/ci/infra/openstack/"
+    dest: "{{ socok8s_workspace }}/skuba-terraform/"
+    mode: 0775
+    force: no
+
+- name: Create a terraform variable file
+  template:
+    src: skuba-terraform.tfvars.j2
+    dest: "{{ socok8s_workspace }}/skuba-terraform/terraform.tfvars"
+  vars:
+    # TODO(toabctl): For now just use the image that the CaaSP team is using
+    image_name: "SLE-15-SP1-JeOS-GM"
+    # NOTE(toabctl):  net/subnet/router variable names must match the names from heat-templates/openstack-network
+    internal_net: "{{ socok8s_envname }}-net"
+    internal_subnet: "{{ socok8s_envname }}-subnet"
+    internal_router: "{{ socok8s_envname }}-router"
+    external_net: "{{ deploy_on_openstack_external_network }}"
+    stack_name: "{{ socok8s_envname }}-caasp4"
+    subnet_cidr: "{{ deploy_on_openstack_internal_subnet_cidr }}"
+    authorized_keys:
+      - "{{ lookup('file', '{{ ansible_env.HOME }}/.ssh/id_rsa.pub', errors='ignore') }}"
```

Comment:
I would prefer exposing this as a var, which could be in the role defaults, with the current value. It would allow users to override it (should they have a different file, or want to ensure failures if absent).

Reply:
Done